### PR TITLE
Zacharyb/pod spec labels

### PR DESCRIPTION
**Pull request by wandb-zacharyblasczyk**
Our helm templating didn't pass labels correctly in all cases. This pr ensures that the labels are correctly applied to the pods. 

Using a values file that has this set:
```yaml
global:
  common:
    labels:
      mls-cluster: wcnp
      mlp-index: mlp_staging_wcnp
prometheus:
  instance:
    commonMetaLabels:
      mls-cluster: wcnp
      mlp-index: mlp_staging_wcnp
```
I got this output:

```bash
kubectl get pods -o custom-columns='NAME:.metadata.name,MLS-CLUSTER:.metadata.labels.mls-cluster,MLP-INDEX:.metadata.labels.mlp-index,WANDB-APP-NAME:.metadata.labels.wandb\.com\/app-name'

NAME                                      MLS-CLUSTER   MLP-INDEX          WANDB-APP-NAME
wandb-app-6fc8bf496c-482zw                wcnp          mlp_staging_wcnp   app-0.1.0
wandb-console-89cdd947d-47w5k             <none>        <none>             console-0.1.0
wandb-otel-agent-mmr4z                    wcnp          mlp_staging_wcnp   otel-0.1.0
wandb-otel-agent-sh822                    wcnp          mlp_staging_wcnp   otel-0.1.0
wandb-parquet-676f88fc9f-n7fz2            <none>        <none>             parquet-0.1.0
wandb-prometheus-server-54b6b85b9-jpv2d   <none>        <none>             <none>
wandb-weave-7d76c7c87d-fwc9s              <none>        <none>             weave-0.1.0
```

### CODE REVIEW

File: charts/operator-wandb/smooth-template.yaml [hunk 1]
```diff
@@ -0,0 +1,2834 @@
+---
```
Comment (nfoucha) [inline charts/operator-wandb/smooth-template.yaml:1]: What purpose does this file serve?  It looks to be autogenerated, so I'd be wary of including it here.

**Review by nfoucha — COMMENTED** [review]
Need some clarification on the smooth manifest, other changes look good

File: charts/operator-wandb/smooth-template.yaml (hunk 1 above)
Comment (wandb-zacharyblasczyk) [reply charts/operator-wandb/smooth-template.yaml]: Yah, that was my bad.

Comment (wandb-zacharyblasczyk) [reply charts/operator-wandb/smooth-template.yaml]: Rotated that license.

**Review by adityachoudhari26 — APPROVED** [review]

**Pull request merged**